crates/threshold-bls-ffi/src/ffi.rs

```diff
@@ -464,7 +464,7 @@ unsafe fn deserialize<T: DeserializeOwned>(
     len: usize,
     out: *mut *mut T,
 ) -> bool {
-    let buf = unsafe { std::slice::from_raw_parts(in_buf, len) };
+    let buf = std::slice::from_raw_parts(in_buf, len);
```

Comment:
We want to keep the amount of unsafe code as tightly scoped as possible. Instead of removing the inner `unsafe` declarations in this function, I'd recommend removing the `unsafe` declaration on the function itself.